### ISSUE TITLE
GET On Session is replacing '//' in url

### ISSUE THREAD

**Issue by nmoreira** [MarketSquare/robotframework-requests#369]
We have a test where we need to send a GET request with a few parameters in the url.
An example of the correct url is: `http://myserver.com/somepath/<parameter1>/<parameter2>`
The test we are running consists in verify some behaviour of the application when parameter1 is missing, thus the url that should be used in our test would be: `http://myserver.com/somepath//<parameter2>`.

We are opening the session correctly to the server `http://myserver.com` and then sending the GET On session passing as argument the session and the url `/somepath//<parameter2>` but the request that is being sent to the server is with the full path `http://myserver.com/somepath/<parameter2>` where the "//" is being replaced with a single "/" making impossible to test the desired scenario.

We are running Robot Framework 6.0.1 (Python 3.8.10 on linux)
and robotframework-requests    0.9.4

Any advice on how to avoid that replacement?

**Comment by lucagiove** [comment]
I believe this is due to the url merge_url code from session:

    def _merge_url(session, uri):
        """
        Helper method that join session url and request url.
        It relies on urljoin that handles quite good join urls and multiple /
        but has some counter intuitive behaviours if you join uri starting with /
        It handles also override in case a full url (http://etc) is passed as uri.
        """
        base = ''
        if session:
            base = session.url
        if session and uri and not session.url.endswith('/'):
            base = session.url + '/'
        if session and uri and uri.startswith('/'):
            uri = uri[1:]
        url = urljoin(base, uri)
        return url

Some ideas I've not time to try myself:
- use a session less request keyword (GET)
- pass an empty params object
- pass a triple slash ///

**Comment by nmoreira** [comment]
Hi @lucagiove! Thanks for your time!

I've spent some time debugging this issue, and as you suspect the double slash disappears in the `urljoin(base, uri)` method.
When you suggest a triple slash, that is in the middle of the uri, like that `/somepath///<parameter2>` right? I've tried and it didn't work.
The empty in params object also didn't work.

I've tried with a double slash at the beginning of the uri, `//somepath//<parameter2>` and to my surprise, it worked. The initial slash gets removed but the double slash in the middle remains untouched. Does it make any sense to you?

For me it works for now as a workaround and I can progress my task, but it seems a bit strange and I'm not convinced that this is the right way to do that.

Again, thanks for your time trying to help.
